Change README to reflect the new changes (#6)
gazebo-web/auth: README.MD

# Auth

> Auth is an open source library for adding authentication and authorization to web services. 

Reviewer: It would be helpful to mention that it is designed with a generic interface and provides some integrations (mention which ones) to allow projects to decide which auth providers they want to use.

Author: Care to give this another pass? I've added a new section called `Providers` that you might want to take a look at.

Author: I can also add another column for the `scheme`; for the auth0 entry it would be `access token`.

# Auth

Auth is an open source library for adding authentication and authorization to web services. 

> It helps developers reducing the amount of time needed to set up an auth layer in their applications, allowing them to focus on developing new features instead.

Reviewer: ```suggestion
Its objective is to reduce the amount of time needed to set up an auth layer in applications, allowing them to focus on developing new features instead.
```
